module5/Publisher.py

```diff
@@ -60,3 +68,12 @@ def pub_builder(pub_type: PublicationType, pub_text: List[str], spec_info: str)
     def writer(pub: Publication) -> None:
```

Comment:
I would rename it in more "actionable" way like write_to_newsfeed

Reply:
Fixed.